src/components/ArticlePreview.jsx

```diff
@@ -46,7 +42,7 @@ export default class ArticlePreview extends BaseComponent {
           <p className="article-preview__content__teaser">{article.teaser}</p>
           <SharingButtons
             title={article.title}
-            url={'thegazelle.org' + url}
+            url={`thegazelle.org${url}`}
```

Comment:
Nope, `<SharingButtons />` inserts this into a URL that already has `https://www.` as part of it.